richie/js/settings.json

```diff
@@ -1,7 +1,10 @@
 {
+  "API_ENDPOINTS": {
+    "COURSE": "/api/v1.0/course"
```

Comment:
I would have preferred to define an `API_ROOT_URL` setting and don't include this root url in endpoints to avoid repetition, but we are limited by the JSON format here. Do you think a plain JS/TS file would better suite our needs for the application's settings.

Reply:
Less repetition in the file itself, but we still need a type definition file along the JS/JSON as we can't import TS into the webpack config

On the other hand JSON is more generally interoperable/reusable later on, and I don't think repetition is necessarily a bad thing in a JSON settings file

Reply:
Okay. Let's try this then.